libgnucash/engine/mocks/gmock-Account.h

```diff
@@ -20,6 +20,9 @@ GType gnc_mock_account_get_type(void);
 class MockAccount : public Account
 {
 public:
+    // note: don't use default constructor instead of empty constructor, since
+    // it does zero initialization, which would overwrite GObject
+    // initialization, which is already done in the new operator.
```

Comment:
Please use `/*...*/` for multi-line comments.

Reply:
Done